backend/src/main/java/de/roman/toolio/security/AppUser.java

```diff
@@ -0,0 +1,33 @@
+package de.roman.toolio.security;
+
+import de.roman.toolio.model.InquiryPart;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+@Document(collection="users")
+public class AppUser {
+    @Id
+    private String id;
+    private String name;
+    private String address;
+    private String email;
+    private List<String> inquiryPartIDs;
+
+    public List<String> getInquiryPartIDs() {
+        if(this.inquiryPartIDs == null){
+            return new ArrayList<>();
+        }
+        return inquiryPartIDs;
+    }
```

Comment:
i would always initialize the list when creating the user instead of writing this functionality